tools/make-config.py

```diff
@@ -340,6 +340,13 @@ def resolve_ovl_export(name: str, ovl_name: str):
     ovl_prefix = make_ovl_name_from_fullname(ovl_name).upper()
     return name.replace("OVL_EXPORT(", f"{ovl_prefix}_").rstrip(")")
 
+##### YAML UTILITIES
+class HexInt(int): pass
+
+def representer(dumper, data):
+    return yaml.ScalarNode('tag:yaml.org,2002:int', hex(data))
```

Comment:
Added this otherwise the default yaml formatter will output it as decimal representation